test/e2e/utils.go

```diff
@@ -135,6 +135,7 @@ func WaitForUserToBeReady(t *testing.T, framework *framework.Framework, namespac
 
 func WaitForSuccessResponseToContain(t *testing.T, framework *framework.Framework, url string, expectedString string) error {
 	return WaitForCondition(t, framework.KubeClient, func(t *testing.T, c kubernetes.Interface) error {
+		//golint:ignore
```

Comment:
Is it OK to suppress the warning? It supposed to be a workaround?

Reply:
Yes. I believe I removed this in my linter fix PR